test/python/expected/sql_validation_framework/expected_create.out

```diff
@@ -103,6 +104,7 @@ Could not find upgrade tests for function sys.ceiling
 Could not find upgrade tests for function sys.columns_internal
 Could not find upgrade tests for function sys.cursor_rows
 Could not find upgrade tests for function sys.cursor_status
```

Comment:
Please add upgrade test for the function.

Reply:
so I am not calling this `date_bucket_internal_helper()` function directly from sqlcmd instead this is a helper function for date_bucket(). Similar to other function that are there for example `datepart_internal()`